src/game/boe.actions.cpp

```diff
@@ -2114,29 +2113,19 @@ bool handle_keystroke(const sf::Event& event){
 }
 
 bool handle_scroll(const sf::Event& event) {
-	rectangle status_panel_rect = {0,0,144,271}, text_panel_rect = {0,0,138,271};
 	rectangle world_screen = win_to_rects[WINRECT_TERVIEW];
 	world_screen.inset(13, 13);
-	status_panel_rect.offset(win_to_rects[WINRECT_INVEN].topLeft());
-	text_panel_rect.offset(win_to_rects[WINRECT_TRANSCRIPT].topLeft());
 	fill_rect(mainPtr, world_screen, sf::Color::Magenta);
+	
+	// XXX TODO centralize this translation somewhere
```

Comment:
Can you please write TODO comments as `// TODO: ...`? Note the colon. (C-style TODO comments are fine too, for the record.)

Reply:
done